app/views/sales/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Create a Sale</h1>
+
+<section>
+  <%= form_for @sale, url: "/vendors/#{:vendor_id}/sales" do |f| %>
```

Comment:
Similar to my comment for the "Add a Product" page, we could probably write this as `form_for [@vendor, @sale] do |f|`, and Rails will figure out the correct route for us.